.vscode/settings.json

```diff
@@ -129,5 +129,35 @@
         "yaml.schemas": {
                 "${workspaceFolder}/validation/module_schema.yaml": "${workspaceFolder}/src/modules/*/module.yaml"
         },
-        "ros.distro": "humble"
+        "ros.distro": "humble",
+        "python.autoComplete.extraPaths": [
```

Comment:
Cleanup to remove local developer configuration.

Reply:
This will be done later - for now, it's a draft.